### PR TITLE
Helm support local kubernetes executor

### PR DESCRIPTION
**Pull request by subkanthi**
Helm changes to support LocalKubernetesExecutor

Airflow Docker images dont have the https://github.com/apache/airflow/pull/19729 changes yet.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -219,7 +219,7 @@ rbac:
   createSCCRoleBinding: false
 
 # Airflow executor
-# Options: LocalExecutor, CeleryExecutor, KubernetesExecutor, CeleryKubernetesExecutor
+# Options: LocalExecutor, LocalKubernetesExecutor, CeleryExecutor, KubernetesExecutor, CeleryKubernetesExecutor
```
Comment (jedcunningham) [inline chart/values.yaml]:
```suggestion
# One of: LocalExecutor, LocalKubernetesExecutor, CeleryExecutor, KubernetesExecutor, CeleryKubernetesExecutor
```

Gets us 1 character down :)

**Comment by jedcunningham** [comment]
Hey @subkanthi, let's get this in the next release so we have day 1 support when 2.3.0 goes out. I think we have a test to fix, otherwise this is ready to go right?

**Comment by subkanthi** [comment]
> Hey @subkanthi, let's get this in the next release so we have day 1 support when 2.3.0 goes out. I think we have a test to fix, otherwise this is ready to go right?
Sure, will fix the test and pull the latest.

File: tests/charts/test_basic_helm_chart.py [hunk 2]
```diff
@@ -327,7 +327,8 @@ def test_unsupported_executor(self):
                 },
             )
         assert (
-            'executor must be one of the following: "LocalExecutor", "CeleryExecutor", '
+            'executor must be one of the following: "LocalExecutor", '
```
Comment (subkanthi) [inline tests/charts/test_basic_helm_chart.py:330]: Assert on the error message maybe is not a good idea

Comment (jedcunningham) [reply tests/charts/test_basic_helm_chart.py:330]: It's a little fragile, yeah, but honestly it changes so infrequently I think it's okay.

**Review by jedcunningham — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Pull request merged**